### PR TITLE
Home Page Changes

### PR DESCRIPTION
**Pull request by Dgarc359**
Add grid layout to homepage for components

### CODE REVIEW

**Review by tsundDEV — APPROVED** [review]

File: src/renderer/app/pages/Main.tsx [hunk 1]
```diff
@@ -1,5 +1,92 @@
...
+        <div style={{ backgroundColor: 'purple', height: '90%', width: '80%' }}>
+          <DashboardContainer />
+        </div>
+        <div style={{ backgroundColor: 'white', height: '10%', width: '100%', opacity: '0.2' }} />
+      </div>
+    </>
+  );
+};
+
 export default function Main() {
```
Comment (jordanjones98) [inline src/renderer/app/pages/Main.tsx:86]: Why is this needed? Why not just default export `Container`?
